Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Dockerfile for a standard freedom-for-node instance 
+
+FROM google/nodejs
```

Comment:
this appears to have git at least - https://registry.hub.docker.com/u/google/nodejs/dockerfile/

does it realy not have a ```node``` or ```npm``` binary?
otherwise you could totally avoid the RUN line.